### PR TITLE
feat: create `InlinePopover` component

### PR DESCRIPTION
**Pull request by sunnyzanchi**
also sets up the `TwoButton` layout (currently the only one), and makes it easy to add more

## testing
- pick your favorite MDX file
- add `<InlinePopover type="apm" />`, or one of the other `type`s in `popovers.json`
- check that page and make sure the popover renders correctly and Tessen events are sent

### CODE REVIEW

File: src/components/InlinePopover/layouts/TwoButton.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  primaryButton,
+  primaryButtonUrl,
+  secondaryButton,
+  secondaryButtonUrl,
+  type,
+  ...props
+}) => {
+  const capsType = capitalize(type);
+  const instrumentation = (eventName) => ({
+    category: `${capsType}Popover`,
```
Comment (tabathadelane) [inline src/components/InlinePopover/layouts/TwoButton.js:26]: Nit: In `InlinePopover.js` we are sending `${type}InlinePopover` Is there a reason these should be different, or is it just an oversight?

Comment (sunnyzanchi) [reply src/components/InlinePopover/layouts/TwoButton.js:26]: ah good catch, no they should be the same

Comment (sunnyzanchi) [reply src/components/InlinePopover/layouts/TwoButton.js:27]: changed this in `InlinePopover.js` to just `${type}Popover`

**Review by clarkmcadoo — APPROVED** [review]

**Comment by tabathadelane** [comment]
I love it 🤩 
Screen Shot 2023-03-08 at 9 53 56 AM

It doesn't love being at the edge of the screen, but i dont know if there is anything we can do about that
Screen Shot 2023-03-08 at 9 57 18 AM

**Comment by sunnyzanchi** [comment]
> It doesn't love being at the edge of the screen when there are no navs, but i dont know if there is anything we can do about that

hmm, i copied over the original logic from the LicenseKey popover which is supposed to avoid that scenario by offsetting the popover left/right. let me double check

**Pull request merged**